engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -125,6 +126,9 @@
     private GraphicState graphicState;
     private PostProcessor postProcessor;
 
+
+    private ShadowMapNode shadowMap;
+    private DirectedAcyclicClassGraph<RenderNode> dag = new DirectedAcyclicClassGraph<>();
```

Comment:
Similar to comment in previous file, let's rename this variable renderingPipeline. Also, we might want to add a comment in both cases saying that this is a work in progress and eventually the whole rendering process will go through this object, not just the ShadowMap. Just in case somebody browses the code in the next few weeks, before the whole thing actually happens. Otherwise it'd be a bit tricky to understand why something like a rendering pipeline is taking care only of a shadow map.